angularjs-1.5-basics/gulpfile.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var gulp = require('gulp');
+var runSequence = require('run-sequence');
+var requireDir = require('require-dir');
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->